rstudio/rhel9-python-3.9/Dockerfile

```diff
@@ -64,6 +64,11 @@ RUN chmod 1777 /var/run/rstudio-server && \
     mkdir -p /usr/share/doc/R
 COPY rstudio/rhel9-python-3.9/rsession.conf /etc/rstudio/rsession.conf
 
+# package installation 
+RUN dnf install -y libsodium-devel.x86_64 libgit2-devel.x86_64 libcurl-devel harfbuzz-devel.x86_64 fribidi-devel.x86_64 cmake "flexiblas-*"
```

Comment:
Not sure why we picked up the `.x86_64` suffixes for some packages. It is not harmful, though, and by the time we'll need to support arm64, this will be the least of our worries :P